output/datadog/chunkidgen.go

```diff
@@ -0,0 +1,33 @@
+package datadog
+
+import (
+	"fmt"
+	"sync"
+	"time"
+)
+
+// TODO: Share this stuff between outputs, since they are duplicates
+
+const chunkIDSuffix = ".ff"
```

Comment:
the chunk id suffix needs to be unique, in case queue dirs of different output types are mixed up by user. It could be something like `.datadog` or `.dd`